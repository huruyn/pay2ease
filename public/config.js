window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";