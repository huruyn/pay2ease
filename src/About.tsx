import { useTranslation } from 'react-i18next';
import AboutUs_Hero from './assets/img/background/AboutUs_Hero.jpg';
import bluestrip from './assets/img/background/bluestrip.jpg';
import Logo_AboutUs from './assets/img/general/Logo_AboutUs.png';
import ZippyCash_Icons_D from './assets/img/roundicons/aboutus/ZippyCash_Icons_D.png';
import ZippyCash_Icons_E from './assets/img/roundicons/aboutus/ZippyCash_Icons_E.png';
import ZippyCash_Icons_F from './assets/img/roundicons/aboutus/ZippyCash_Icons_F.png';
import ZippyCash_Icons_G from './assets/img/roundicons/aboutus/ZippyCash_Icons_G.png';
import Footer from './footer/Footer';

export default function About() {
  const { t } = useTranslation();

  return (
    <>
      <main>
        <div className="container">
          <div>
            <div className="row">
              <div className="col-12 col-md-5 col-lg-4">
                <br />
                <br />
                <h2 className="redsubtitle">{t('about.who_is')}</h2>
                <img src={Logo_AboutUs} alt="zippy.cash?" className="img-fluid" />
                <br />
                <br />
                <p className="content-home" style={{ fontWeight: 600 }}>
                  {t('about.who_is_description_1')}
                </p>
                <br />
                <p className="content-home">
                  {t('about.who_is_description_2')}
                </p>
                <div />
              </div>
              <div className="col-12 col-md-7 col-lg-8" style={{ backgroundImage: `url(${AboutUs_Hero})`, backgroundSize: 'cover' }}>
                <div />
              </div>
            </div>
            <div className="row">
              <div style={{ backgroundImage: `url(${bluestrip})`, backgroundRepeat: 'repeat-x' }}>
                <div className="row mt-4">
                  <div className="col-12 col-md-6 col-lg-3">
                    <div className="mb-4 aboutus_icon">
                      <img
                        src={ZippyCash_Icons_D}
                        alt="styled icon of a man holding a star"
                        className="img-fluid centerimage"
                        data-aos="flip-left"
                      />
                    </div>
                    <div>
                      <h4 className="redtitle text-center">{t('about.the_team')}</h4>
                      <p className="content-home">{t('about.the_team_description')}</p>
                    </div>
                  </div>
                  <div className="col-12 col-md-6 col-lg-3">
                    <div className="mb-4 aboutus_icon">
                      <img
                        src={ZippyCash_Icons_E}
                        alt="styled icon of a man holding a light bulb"
                        className="img-fluid  centerimage"
                        data-aos="flip-left"
                        data-aos-delay="200"
                      />
                    </div>
                    <div>
                      <h4 className="redtitle text-center">{t('about.our_mission')}</h4>
                      <p className="content-home">{t('about.our_mission_description')}</p>
                    </div>
                  </div>
                  <div className="col-12 col-md-6 col-lg-3">
                    <div className="mb-4 aboutus_icon">
                      <img
                        src={ZippyCash_Icons_F}
                        alt="styled icon of a woman holding a heart"
                        className="img-fluid  centerimage"
                        data-aos="flip-left"
                        data-aos-delay="400"
                      />
                    </div>
                    <div>
                      <h4 className="redtitle text-center">{t('about.our_values')}</h4>
                      <p className="content-home">{t('about.our_values_description')}</p>
                    </div>
                  </div>
                  <div className="col-12 col-md-6 col-lg-3">
                    <div className="mb-4 aboutus_icon">
                      <img
                        src={ZippyCash_Icons_G}
                        alt="styled icon of a person holding a mechnical gear"
                        className="img-fluid  centerimage"
                        data-aos="flip-left"
                        data-aos-delay="600"
                      />
                    </div>
                    <div>
                      <h4 className="redtitle text-center">{t('about.our_partner')}</h4>
                      <p className="content-home">{t('about.our_partner_description')}</p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      <Footer />
    </>
  );
}
