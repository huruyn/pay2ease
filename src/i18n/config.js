import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;