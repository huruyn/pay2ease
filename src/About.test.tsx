import './i18n/config';
import About from './About';

const ReactTestRenderer = require('react-test-renderer');

describe('About Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
