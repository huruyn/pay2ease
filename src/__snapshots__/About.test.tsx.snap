// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component matches the snapshot 1`] = `
Array [
  <main>
    <div
      className="container"
    >
      <div>
        <div
          className="row"
        >
          <div
            className="col-12 col-md-5 col-lg-4"
          >
            <br />
            <br />
            <h2
              className="redsubtitle"
            >
              who is
            </h2>
            <img
              alt="zippy.cash?"
              className="img-fluid"
              src="Logo_AboutUs.png"
            />
            <br />
            <br />
            <p
              className="content-home"
              style={
                Object {
                  "fontWeight": 600,
                }
              }
            >
              We're finance geeks who know the system. And we think it's too complicated, too expensive and too difficult. Just because it's always been that way doesn't mean we can't do better. We're passionate about working the system to make it rewarding for everyone. We created Zippy Cash to deliver on that promise.
            </p>
            <br />
            <p
              className="content-home"
            >
              In a fair world, transparent and affordable financial services would be accessible to all, regardless of who they are. But… you need a law degree to understand the fine print and in the end, consumers end up getting ripped off - especially those who have less. Zippy Cash exists to break these chains and make easy, fast, secure payments a reality for anyone who wants them!
            </p>
            <div />
          </div>
          <div
            className="col-12 col-md-7 col-lg-8"
            style={
              Object {
                "backgroundImage": "url(AboutUs_Hero.jpg)",
                "backgroundSize": "cover",
              }
            }
          >
            <div />
          </div>
        </div>
        <div
          className="row"
        >
          <div
            style={
              Object {
                "backgroundImage": "url(bluestrip.jpg)",
                "backgroundRepeat": "repeat-x",
              }
            }
          >
            <div
              className="row mt-4"
            >
              <div
                className="col-12 col-md-6 col-lg-3"
              >
                <div
                  className="mb-4 aboutus_icon"
                >
                  <img
                    alt="styled icon of a man holding a star"
                    className="img-fluid centerimage"
                    data-aos="flip-left"
                    src="ZippyCash_Icons_D.png"
                  />
                </div>
                <div>
                  <h4
                    className="redtitle text-center"
                  >
                    THE TEAM
                  </h4>
                  <p
                    className="content-home"
                  >
                    We're finance and tech geeks who are passionate about the freedom of choice the digital world has made possible. We love other out-of-the-box thinkers and FinTech wizards so if that's you, check out Zippy Cash employment opportunities here.
                  </p>
                </div>
              </div>
              <div
                className="col-12 col-md-6 col-lg-3"
              >
                <div
                  className="mb-4 aboutus_icon"
                >
                  <img
                    alt="styled icon of a man holding a light bulb"
                    className="img-fluid  centerimage"
                    data-aos="flip-left"
                    data-aos-delay="200"
                    src="ZippyCash_Icons_E.png"
                  />
                </div>
                <div>
                  <h4
                    className="redtitle text-center"
                  >
                    OUR MISSION
                  </h4>
                  <p
                    className="content-home"
                  >
                    Our mission is to level the playing field and reward customers for doing the things they do every day - even the small stuff. We do this through expertise, partnerships and best-in-class tech that allows us to power up your money - faster, easier, safer... and fun too - that's Zippy!
                  </p>
                </div>
              </div>
              <div
                className="col-12 col-md-6 col-lg-3"
              >
                <div
                  className="mb-4 aboutus_icon"
                >
                  <img
                    alt="styled icon of a woman holding a heart"
                    className="img-fluid  centerimage"
                    data-aos="flip-left"
                    data-aos-delay="400"
                    src="ZippyCash_Icons_F.png"
                  />
                </div>
                <div>
                  <h4
                    className="redtitle text-center"
                  >
                    OUR VALUES
                  </h4>
                  <p
                    className="content-home"
                  >
                    We believe our customers deserve the utmost in security and transparency. We believe in choices for everyone, regardless of how big your bank account is. We believe in rewarding our customers every day, because we know a little more money can give folks a lot more power. And it's about time that power was accessible to everyone.
                  </p>
                </div>
              </div>
              <div
                className="col-12 col-md-6 col-lg-3"
              >
                <div
                  className="mb-4 aboutus_icon"
                >
                  <img
                    alt="styled icon of a person holding a mechnical gear"
                    className="img-fluid  centerimage"
                    data-aos="flip-left"
                    data-aos-delay="600"
                    src="ZippyCash_Icons_G.png"
                  />
                </div>
                <div>
                  <h4
                    className="redtitle text-center"
                  >
                    OUR PARTNERS
                  </h4>
                  <p
                    className="content-home"
                  >
                    We work hand in hand with government bodies and financial industry partners to ensure our customers get the best of the best. Check out who's on our roster.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>,
]
`;
